attribute/document_details_attribute_test.go

```diff
@@ -24,6 +25,24 @@ func ExampleDocumentDetails_Parse() {
 	// Output: Document Type: PASSPORT, Issuing Country: GBR, Document Number: 1234567, Expiration Date: 2022-09-12 00:00:00 +0000 UTC
 }
 
+func ExampleNewDocumentDetails() {
+	proto := yotiprotoattr.Attribute{
+		Name:        "exampleDocumentDetails",
+		Value:       []byte("PASSPORT GBR 1234567 2022-09-12"),
+		ContentType: yotiprotoattr.ContentType_STRING,
+	}
+	attribute, err := NewDocumentDetails(&proto)
+	if err != nil {
+		return
+	}
+	fmt.Printf(
+		"Document Type: %s, With %d Anchors",
+		attribute.Value().DocumentType,
+		len(attribute.Anchors()),
+	)
+	// Output: Document Type: PASSPORT, With 0 Anchors
```

Comment:
Is there a reason we didn't use asserts here? 
e.g. 
```go
	assert.Equal(t, "PASSPORT", attribute.Value().DocumentType)
	assert.Equal(t, 0, len(attribute.Anchors()))
```

I think it make it a bit clearer what we're actually testing

Reply:
You can't use asserts in Examples.

Examples get compiled into the autgenerated documentation, so it's good to add them to Public methods.

Reply:
Ah ok, gotcha